### ISSUE TITLE
Iconos para redes sociales

### ISSUE THREAD

**Issue by UlisesGascon** [GingerCode/Art#4]
Hola @ancoar!

¿Podrías añadir crear iconos para redes sociales (circulares y rectangulares) en los formatos y extensiones típicas?

Muchas gracias!!

**Comment by ancoar** [comment]
Estoy en ello… 👍 Pero creo que lo suyo es que los iconos sean SVG y solo SVG… ¿Vamos a hacer las cosas bien no? 😋 Pues lo suyo sería que fuesen en SVG e inyectados inline en el HTML como símbolos para luego instanciar con `<use xlink:href="#xxxxxx"></svg>` donde queramos los enlaces icono.

**Comment by UlisesGascon** [comment]
Correcto! La idea es usar vectorial siempre que se pueda, pero algunas plataformas no lo.sopoetan aún :)

**Comment by ancoar** [comment]
Holaaa ok!, si no tengo problema en exportar a PNG también. Será un instante 😊. Lo decía porque creo que Electron funciona con Chromium, por lo que no nos harían falta fallbacks no?
